src/bindings/interface/importsource.i

```diff
@@ -31,6 +31,36 @@ unset).";
 %feature("docstring") libcellml::ImportSource::clone
 "Create a copy of this import source.";
 
+%feature("docstring") libcellml::ImportSource::addComponent
+"Add a dependent Component to this ImportSource.";
```

Comment:
I would remove "dependent".

Reply:
Also, would use "component" not "Component". (This is what we do elsewhere in our interfaces.) Same below.

Reply:
Why? It has knock-on effects for the component, so it is a dependent component?

Reply:
Same with "ImportSource" which should be replaced with "import source".

Reply:
Just because the method is called `addComponent`. If it was called `addDependentComponent` (which I wouldn't want) then it would make sense for me to have "dependent", but not as it stands. Just my 2c.

Reply:
Eh.  I'd hope that docstrings were allowed to have more info than just rewriting their function name!  I've changed it for now 7a544255 but hopefully #618 can improve what we provide to users.